enterprise/cli/testdata/coder_server_--help.golden

```diff
@@ -327,6 +327,68 @@ can safely ignore these settings.
           Minimum supported version of TLS. Accepted values are "tls10",
           "tls11", "tls12" or "tls13".
 
+NOTIFICATIONS OPTIONS: 
```

Comment:
Thinking loud: maybe you can start with max 3 options and consider others to be hardcoded for now?